skorch/tests/test_helper.py

```diff
@@ -798,7 +798,13 @@ def accelerator_cls(self):
         pytest.importorskip('accelerate')
 
         from accelerate import Accelerator
+        from accelerate.state import AcceleratorState
 
+        # We have to use this private method because otherwise, the
+        # AcceleratorState is not completely reset, which results in an error
+        # initializing the Accelerator more than once in the same process.
+        # pylint: disable=protected-access
+        AcceleratorState._reset_state()
```

Comment:
Using `AcceleratorState` to influence `Accelerator` is weird from a software design point of view. Is there a public API to reset the `Accelerator`?

Seeing the test changes in https://github.com/huggingface/accelerate/pull/458, I suspect there is no public API.

Reply:
Yes, I agree that it's unfortunate that this has to be done, and the fact that it's necessary in the first place is surprising as a user. The devs told me this is the way to go. Maybe there will be a better solution in the future.